library/src/components/shared/Button/Button.styles.tsx

```diff
@@ -0,0 +1,25 @@
+import Button, { ButtonProps } from '@mui/material/Button';
```

Comment:
```suggestion
import { ButtonProps } from '@mui/material/Button';
import Button from './Button';
```
Remove Button.tsx otherwise